### PR TITLE
Ooo case insensitive

### PR DESCRIPTION
**Pull request by deanna-abrams**

### CODE REVIEW

File: src/components/graphql/UserIssues.tsx [hunk 1]
```diff
@@ -8,14 +8,14 @@ import { projects } from 'data/projects';
 import { organization } from 'data/customize';
 
 import { useQuery, useMutation, useApolloClient } from '@apollo/client';
-import { ALL_ISSUES, 
+import { ALL_ISSUES,
```
Comment (deanna-abrams) [inline src/components/graphql/UserIssues.tsx:11]: my lint-on-save went ham on this file

File: src/components/graphql/UserIssues.tsx [hunk 2]
```diff
@@ -192,22 +192,22 @@ const UserIssues: React.FC<UserIssuesProps> = (props: UserIssuesProps) => {
       const standardAndCustomLabels: string[] = labelsForTeam.concat(standardLabels);
 
       // Are there any labels based on the text detected in the issue title?
-      const titleGeneratedLabels = update.title?.indexOf('OOO') >= 0 ? ['OOO 🌴'] : [];
+      const titleGeneratedLabels = update.title?.toUpperCase().indexOf('OOO') >= 0 ? ['OOO 🌴'] : [];
```
Comment (deanna-abrams) [inline src/components/graphql/UserIssues.tsx:195]: this adds the ooo label regardless of case used in creating the issue

File: src/data/actions.ts [hunk 3]
```diff
@@ -3,16 +3,16 @@ import { primaryLabelForIssue } from './labels';
...
 const updateForAction = (action: any, labelToRemove: any) => {
-  
+
   // Compose our label names
   const labelNameToAdd = `${action.name} ${action.icon}`
   const labelNameToRemove = `${labelToRemove.name} ${labelToRemove.icon}`;
 
   // Handle different actions differently
   if ( action.icon === '✅' ) {
-    const removeWhenClosing = ['Doing ⏳', 'Review 👓', 'Blocked 🛑', 'Paused ⏸', ...[labelNameToRemove]];
```
Comment (deanna-abrams) [inline src/data/actions.ts:13]: this was causing priority labels and OOO labels to be removed when an issue was marked as done without going through doing first

**Comment by delaurentis** [comment]
Thanks Deanna!  I hand merged both of your fixes into my latest PR (since I had a bunch of changes to the same files):
https://github.com/delaurentis/planner/pull/21

**Pull request closed without merging**